examples/stable/mysql/mysql-blueprint.yaml

```diff
@@ -57,7 +57,7 @@ actions:
         - |
           s3_path="{{ .ArtifactsIn.mysqlCloudDump.KeyValue.s3path }}"
           root_password="{{ index .Phases.restoreFromBlobStore.Secrets.mysqlSecret.Data "mysql-root-password" | toString }}"
-          kando location pull --profile '{{ toJson .Profile }}' --path ${s3_path} - | gunzip | mysql -u root --password=${root_password} -h {{ .Deployment.Name }}
+          kando location pull --profile '{{ toJson .Profile }}' --path ${s3_path} - | gunzip | mysql -u root --password=${root_password} -h {{ index .Object.metadata.labels "release" | toString }}
```

Comment:
do we not have access to mysql service using `StatefulSet.Name`

Reply:
The service is named after release name, while statefulset is named as "release-name"-master.